Tests/SwiftProtobufTests/Test_Naming.swift

```diff
@@ -184,3 +184,165 @@ class Test_FieldNamingInitials: XCTestCase {
     msg.clearTheURL()
   }
 }
+
+class Test_ExtensionNamingInitials_MessageScoped: XCTestCase {
+  func testLowers() {
+    var msg = SwiftUnittest_Names_ExtensionNamingInitials()
+
+    msg.SwiftUnittest_Names_Lowers_http = 1
```

Comment:
Ok, I'll go ahead and land this.  We can always revisit it we change our mind (before 1.0 is done), it just caused me some pause as I wrote the more complete tests.